aspnetcore/host-and-deploy/linux-nginx.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Host ASP.NET Core on Linux with Nginx
 author: rick-anderson
-description: Learn how to set up Nginx as a reverse proxy on an Ubuntu 20.04 VM to forward HTTP traffic to an ASP.NET Core web app running on Kestrel.
+description: Learn how to set up Nginx as a reverse proxy on Ubuntu 20.04 to forward HTTP traffic to an ASP.NET Core web app running on Kestrel.
```

Comment:
Should we list the distros here too? RHEL/SUSE.

Reply:
Yes!  Fixed meta description, thanks.  I also took out the version in the meta data description since several versions apply.